src/state/selectors.js

```diff
@@ -20,10 +27,16 @@ export const getQuestionsList = createSelector(getQuestions, objectToList);
 export const getResponsesList = createSelector(getResponses, objectToList);
 export const getContentFieldsData = createSelector(getContentFields, listToObject('field-id (don\'t change!)'));
 
-// This app makes an assumption only Emoji questions will be used to group_by
 export const getEmojiQuestion = createSelector(
-  getQuestionsList,
-  questions => questions.find(question => question.group_by)
+  getQuestions,
+  getFilterQuestions,
+  (questions, filterQuestions) => questions && questions[filterQuestions.emoji]
+);
```

Comment:
Storing the IDs of the filter questions in the store lets this become a lookup, not a search